CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -273,11 +273,15 @@ end
 
 function Methods.ResetFrameDimensions(devConsole)
 	devConsole.Frame.Size = UDim2_new(0.5, 20, 0.5, 20);
-	devConsole.Frame.Position = UDim2_new(0.25, -10, 0.125, -10)
+	
+	local abSize = devConsole.Frame.AbsoluteSize
+	devConsole:SetFrameSize(abSize.x, abSize.y)
```

Comment:
I should also mention that I added code to SetFrameSize to check if it's too small.